main.go

```diff
@@ -204,9 +205,15 @@ func NewExporter(server string, timeout time.Duration, server_stats bool, logger
 			nil,
 			nil,
 		),
-		clients: prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, "", "clients"),
-			"Number of connected clients.",
+		numClients: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "num_clients"),
```

Comment:
The old prometheus metric name is changed here so won't be backwards compatible? This should be `clients`  still.

Reply:
Updated to keep the metric name the same